mesa/visualization/templates/modular_template.html

```diff
@@ -80,22 +81,24 @@ <h4 class="modal-title">About {{ model_name }}</h4>
     </div>
 
     <!-- Bottom-load all JavaScript dependencies -->
-    <script src="/static/external/bootstrap-5.1.3-dist/js/bootstrap.bundle.min.js"></script>
-    <script src="/static/external/bootstrap-slider-11.0.2/dist/bootstrap-slider.min.js"></script>
+    <script src="./static/js/external/jquery-2.2.4.min.js"></script>
```

Comment:
Those are old versions of the libraries. Also, why are 4 lines added instead of just 2?